src/components/ble/MusicService.cpp

```diff
@@ -18,6 +18,7 @@
 #include "components/ble/MusicService.h"
 #include "components/ble/NimbleController.h"
 #include <cstring>
+#include <task.h>
```

Comment:
You need to include `FreeRTOS.h` before `task.h`.

Reply:
It's being provided by nimblecontroller.h (at least by fs.h), but I suppose it's better to avoid the indirect dependency?

Reply:
Yeah. I'm of the opinion that all of our headers should work regardless of how they're included in cpp files.

Reply:
I think that does make sense, that way it's tolerant to things being moved around later. This probably does mean that a lot of headers are missing across the project though

Reply:
I don't think we need to fix it for existing code, but going forward we should aim for it on a best-effort basis.

Reply:
added FreeRTOS.h include before tasks.h now